server/integration/feature-pack/src/main/resources/content/bin/prometheus.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ -z "$JMX_CONF" ]
+then
+   if [ -z "$1" ]
+   then
+       echo "Neither JMX_CONF nor input parameters are defined. Exiting"
+       exit 1
+   else
+       JMX_CONF=$1
+   fi
+fi
+
+PROMETHEUS_JAR=$(ls $JBOSS_HOME/modules/system/add-ons/ispn/io/prometheus/jmx/ispn-9.4/*.jar | sed -e "s| |:|g")
```

Comment:
You cannot use `*.jar` in these commands as it breaks when utilising the "thin" server. The thin server does not store the jar files in the slot directory, instead it pulls them from the local maven repository.

An alternative solution is todo something like:
```
PROMETHEUS_JAR=jmx_prometheus_javaagent-@prometheus_version@.jar
```
with the value of `@prometheus_version@` set in `server/integration/feature-pack/build.xml`.

To test future fixes, make sure you try running the thin server at: `server/integration/build/target/server-distribution-thin/infinispan-server-9.4.1-SNAPSHOT`

Reply:
@ryanemerson ok, IIRC correctly Tristan and I talked about being able to run the server in the fat mode with Prometheus...but sure, let's make it work in the thin as well.

Reply:
But @ryanemerson I won't be able to pull up two other jars (logmanager, common) in thin server  either. I need these three jars. But thin server will work in non jmx mode which is ok. @tristantarrant said we run thin server for tests only and that fat version matters. Thoughts?

Reply:
Well we distribute the thin server via maven, so I would say it should work.

Another solution, which is probably the way to go, is to not use placeholders. Instead process the `module.xml` of the three modules in order to retrieve the jar name. It is specified in the following format:

`<artifact name="org.wildfly.common:wildfly-common:1.4.0.Final"/>```

Reply:
Sorry, just realised that still doesn't solve the problem of getting the absolute path to the jar

Reply:
Yeah, I think Tristan thought this through as well and concluded that we can only run JMX in fat server mode...unless there is some way we can reference these jars

Reply:
Yeah fair enough, fat server is sufficient for OS which is the primary target anyway.

Reply:
Ok, then the last commit fix should be sufficient. Please check @ryanemerson